skumm00-proj.md

```diff
@@ -0,0 +1,37 @@
+---
+name: "@Skumm00"
+project: "Robot Arm"
+---
+
+#  Project Name - ArduinArm
+
+##  Summary
+
+My project is going to be a custom-controlled, Arduino-powered, robot arm.
+
+##  Plan
+
+I will be using a pre-existing set of 3d models for the overall structure of the robot arm. For the first 3 axes, the waist, the shoulder, and the elbow, 
+I will be using the MG996R servos - for the wrist roll/pitch I will use the smaller SG90 micro servos. The servos will come with screws for installation. 
+The circuitry for this project is simple, with 6 servo motors being connected to six digital pins on the Arduino board. The servos will require at least 5V
+of power coming from the external power source. As for the software behind the arm, there are a number of tutorials I am planning on following (such as this 
+one's servo arm code: https://create.arduino.cc/projecthub/ryanchan/simple-programmable-robotic-arm-bd28a0). The overall code will be a perpetual loop 
+that constantly checks for outside input via bluetooth to the arm - if there is any, the corresponding motor will move. I will likely be using MIT App 
+Inventor in order to create the app to communicate with the robot.
+
+ARM MODEL: https://thangs.com/designer/m/3d-model/38899
+
+##  Budget
+
+| Product  | Supplier/Link  | Cost  |
+| --------------- | ------------------------------------- | ------ |
+| 3D printer  | [Amazon](https://www.amazon.com/Voxelab-Structure-Certified-Removable-8-66x8-66x9-84in/dp/B09BNG5884/ref=sr_1_8?crid=51ZBDXLXFTJD&keywords=3d%2Bprinter&qid=1671945440&sprefix=3d%2Bprint%2Caps%2C148&sr=8-8&ufe=app_do%3Aamzn1.fos.f5122f16-c3e8-4386-bf32-63e904010ad0&th=1) | $159.99  |
+| Arduino UNO | [Amazon](https://www.amazon.com/Arduino-A000066-ARDUINO-UNO-R3/dp/B008GRTSV6/ref=as_li_ss_tl?keywords=arduino+uno&qid=1580073330&sr=8-3&linkCode=sl1&tag=howto045-20&linkId=2de9a9cd18cdb5d608fbd42ba75f1298&language=en_US)  | $28.50 |
```

Comment:
Do you have a USB B cable you can use? It looks like the Arduino linked does not come with one.